proposed/extended-coding-style-guide-meta.md

```diff
@@ -69,8 +65,55 @@ PSR-N will also not change anything stipulated in PSR-1 and PSR-2.
 The overarching approach is to attempt to apply existing PSR-2 styling and rationale to
 new functionality as opposed to establishing new conventions.
 
-
-5. People
+### Strict Types Declarations
+
+There was a discussion about whether or not strict types should be enforced in the standard
+https://github.com/cs-extended/fig-standards/issues/7. All were in agreement we should only
+use a MUST or MUST NOT statement and avoid the use of a SHOULD statement and nobody wanted
+to say that strict types could not be declared. The discussion was whether it should be
+considered a coding style item which should be covered or whether it was out of scope and it
+was decided to be out of scope of a coding style guide.
+
+## Finally and return types declartion spacing
+
+Numerous different options were suggested and they can be seen
+[here for return type declarations](https://gist.github.com/michaelcullum/c025f3870c9ea1dd2668#file-returntypesspacing-php) or
+[here for finally blocks](https://gist.github.com/michaelcullum/c025f3870c9ea1dd2668#file-finallyblocks-php)
+and the current implementation was chosen due to consistency with other parts of the PSR-12
+specification that came from PSR-2.
+
+5. Changelog from PSR-2
+------------------------
+
+Please note this changelog is not a verbose list of changes from PSR-2 but highlights
+most of the most notable changes. It should be considered a new specification and therefore
```

Comment:
I think it should be "most of the most notable changes" > "the most notable changes" so that it reads

> ... but highlights the most notable changes.

instead of 

> ... but highlights most notable changes.